stdlib/LinearAlgebra/test/svd.jl

```diff
@@ -139,4 +139,35 @@ aimg  = randn(n,n)/2
     end
 end
 
+
+
+@testset "SVD Algorithms" begin
+    ≊(x,y) = isapprox(x,y,rtol=1e-15)
+
+    allpos = (v) -> begin
```

Comment:
I am not sure you need a custom function for this, eg
```julia
all(sx1.S .≥ 0)
```
is fine for tests.

Reply:
Alright.